app/lib/features/home/widgets/my_spaces_section.dart

```diff
@@ -81,11 +83,22 @@ class MySpacesSection extends ConsumerWidget {
   }
 }
 
-class _NoSpacesWidget extends ConsumerWidget {
+class _NoSpacesWidget extends ConsumerStatefulWidget {
   const _NoSpacesWidget();
 
   @override
-  Widget build(BuildContext context, WidgetRef ref) {
+  ConsumerState<ConsumerStatefulWidget> createState() => _NoSpacesWidgetState();
+}
+
+class _NoSpacesWidgetState extends ConsumerState<_NoSpacesWidget> {
+  @override
+  void initState() {
+    super.initState();
+    if (!isDesktop) createOrJoinSpaceTutorials(context: context);
```

Comment:
why aren't we also showing that on desktop?

Reply:
We have different approaches of home screen in desktop and mobile.

On desktop, we are showing dashboard as first tab when we start app.
On mobile, we are showing update as first tab when we start app.

So `create and join button` information tutorial need to manage separately based on the platform to avoid conflicting it with `navigation tab buttons` tutorials.

On desktop, we are already showing  `create and join button` information tutorial right after `navigation tab buttons` tutorials with the help of below code.

```
bottom_navigation_tutorials.dart
void showCreateOrJoinSpaceTutorials(BuildContext context) {
  if (isDesktop) createOrJoinSpaceTutorials(context: context);
}
```

Let me know in case of any confusion. :)